scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -384,11 +384,28 @@ private static void CreateDummyRuntimeNuGetPackage(DotNetCli dotnet, string base
                 .WorkingDirectory(tempPjDirectory)
                 .Execute()
                 .EnsureSuccessful();
-
-            dotnet.Build(tempPjFile, "--runtime", rid)
+            dotnet.Build(tempPjFile, "--runtime", "win7-x64")
                 .WorkingDirectory(tempPjDirectory)
                 .Execute()
                 .EnsureSuccessful();
+        }
+
+        private static void CreateDummyRuntimeNuGetPackage(DotNetCli dotnet, string basePackageId, string rid, string version, string outputDir)
+        {
+            var packageId = $"runtime.{rid}.{basePackageId}";
+
+            var projectJson = new StringBuilder();
+            projectJson.Append("{");
+            projectJson.Append($"  \"version\": \"{version}\",");
+            projectJson.Append($"  \"name\": \"{packageId}\",");
+            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24008\" },");
+            projectJson.Append("  \"frameworks\": { \"netcoreapp1.0\": { \"imports\": [\"netstandard1.5\", \"dnxcore50\"] } },");
+            projectJson.Append("}");
+
+            var tempPjDirectory = Path.Combine(Dirs.Intermediate, "dummyNuGetPackageIntermediate");
+            var tempPjFile = Path.Combine(tempPjDirectory, "project.json");
+
+            File.WriteAllText(tempPjFile, projectJson.ToString());
 
             dotnet.Pack(
```

Comment:
Do we even need to pack each project?  What if we just made 1 unplug and then copied the same file 7 times for all the other rids?